File: src/_03_BehavioralPattern/_03_01_ChainOfResponsibilities/before/Request.java
package _03_BehavioralPattern._03_01_ChainOfResponsibilities.before;

public class Request {

  private String body;

  public Request(String body) {
    this.body = body;
  }

  // body
  public String getBody() {
    return this.body;
  }
  public void setBody(String body) {
    this.body = body;
  }
  
}

File: src/_03_BehavioralPattern/_03_01_ChainOfResponsibilities/Request.js
export default class Request {
  /** @type { string } */
  #body;

  /** @param { string } body */
  constructor(body) {
    this.#body = body;
  }

  /** @returns { string } */
  getBody() {
    return this.#body;
  }
  /** @param { string } body */
  setBody(body) {
    this.#body = body;
  }
}